benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -231,6 +272,12 @@ void GraphicsBenchmarkApp::Setup()
 
         mPerFrame.push_back(frame);
     }
+
+    {
+        OffscreenFrame frame = {};
+        PPX_ASSERT_MSG(CreateOffscreenFrame(frame, RenderFormat(), GetSwapchain()->GetDepthFormat(), GetSwapchain()->GetWidth(), GetSwapchain()->GetHeight()) == ppx::SUCCESS, "create offscreen frame buffer");
```

Comment:
Is there a particular reason this uses `PPX_ASSERT_MSG` rather than `PPX_CHECKED_CALL`?

Reply:
Done.